lib/puppet/type/cisco_ace.rb

```diff
@@ -291,4 +291,12 @@ def name
     desc 'Log matches against this entry'
     newvalues(:true, :false)
   end
+
+  validate do
+    unless self[:remark].nil?
+      fail ArgumentError,
+           'established and log parameters must be nil when remark is not nil' unless
```

Comment:
*nit:* I would modify this to read:
`'established' and 'log' properties should not be set for `remark` ace.`